util/update_quick_index.py

```diff
@@ -241,7 +245,15 @@ def process_directory(directory: Path, quick_index_file):
             api_file.write(f"\n\n")
             api_file.write(f"{title}\n")
             underline = "-" * len(title)
-            api_file.write(f"{underline}\n\n")
+            api_file.write(f"{underline}\n")
+
+            # Use automodule to add package level doc string
+            if api_file_name in include_module_docs:
+                api_file.write(
+                    f".. automodule:: {include_module_docs[api_file_name]}\n"
+                )
+                api_file.write("    :no-index:\n")
+            api_file.write(f"\n")
```

Comment:
@pvcraven this would add module doc_string between title and the first members, this is for now only activated for `arcade.gui.experimental`.

the `no-index` prevents double documented code which is listed within the module via `__all__`

Reply:
Removed from this PR, we start without that.